ocm_python_wrapper/cluster.py

```diff
@@ -249,33 +250,69 @@ def addon_info(self):
             self.addon_name
         ).to_dict()
 
-    def validate_addon_parameters(self, parameters):
+    def drop_unsupported_parameters(self, parameters=[]):
+        """ "Filters out addon parameter by conditions"""
+        filtered_parameters = []
+        for param in parameters:
+            for condition in param["conditions"]:
```

Comment:
this does not check multiple conditions. e.g: ` {'aws.sts.enabled': True, 'cloud_provider.id': 'aws'}`

Reply:
Checked in recent update